### PR TITLE
document release and rebuild token triggers

### PR DESCRIPTION
**Pull request by adrianschroeter**

### CODE REVIEW

File: xml/obs_authorization_token.xml [hunk 1]
```diff
@@ -84,6 +84,16 @@
    <screen><command>osc</command> token --create &lt;PROJECT&gt; &lt;PACKAGE&gt;</screen>
    <screen><command>osc</command> token --trigger &lt;TOKEN&gt;</screen>
    <screen><command>curl</command> -H "Authorization: Token &lt;TOKEN&gt;" -X POST /trigger/runservice</screen>
+
+   <para>A token to rebuild a package can be created and trigger by </para>
+   <screen><command>osc</command> token --opertion rebuild --create &lt;PROJECT&gt; &lt;PACKAGE&gt;</screen>
+   <screen><command>osc</command> token --operation rebuild --trigger &lt;TOKEN&gt;</screen>
+   <screen><command>curl</command> -H "Authorization: Token &lt;TOKEN&gt;" -X POST /trigger/rebuild</screen>
+
+   <para>A token to release a package can be created and trigger by </para>
```
Comment (tomschr) [inline xml/obs_authorization_token.xml:93]: Same here, I'd start with a verb:

```suggestion
   <para>To release a package, create a token as follows: </para>
```

File: xml/obs_authorization_token.xml [hunk 2]
```diff
@@ -84,6 +84,16 @@
    <screen><command>osc</command> token --create &lt;PROJECT&gt; &lt;PACKAGE&gt;</screen>
    <screen><command>osc</command> token --trigger &lt;TOKEN&gt;</screen>
    <screen><command>curl</command> -H "Authorization: Token &lt;TOKEN&gt;" -X POST /trigger/runservice</screen>
+
+   <para>A token to rebuild a package can be created and trigger by </para>
+   <screen><command>osc</command> token --opertion rebuild --create &lt;PROJECT&gt; &lt;PACKAGE&gt;</screen>
+   <screen><command>osc</command> token --operation rebuild --trigger &lt;TOKEN&gt;</screen>
+   <screen><command>curl</command> -H "Authorization: Token &lt;TOKEN&gt;" -X POST /trigger/rebuild</screen>
```
Comment (tomschr) [inline xml/obs_authorization_token.xml:91]: There is no need to have separate screens. Better combine all the lines into one. Also use `replaceable`:

```suggestion
   <screen><command>osc</command> token --opertion rebuild --create <replaceable>PROJECT</replaceable> <replaceable>PACKAGE</replaceable>
<command>osc</command> token --operation rebuild --trigger <replaceable>TOKEN</replaceable>
<command>curl</command> -H "Authorization: Token <replaceable>TOKEN</replaceable>" -X POST /trigger/rebuild</screen>
```

File: xml/obs_authorization_token.xml [hunk 3]
```diff
@@ -84,6 +84,16 @@
...
+
+   <para>A token to release a package can be created and trigger by </para>
+   <screen><command>osc</command> token --opertion release --create &lt;PROJECT&gt; &lt;PACKAGE&gt;</screen>
+   <screen><command>osc</command> token --operation release --trigger &lt;TOKEN&gt;</screen>
+   <screen><command>curl</command> -H "Authorization: Token &lt;TOKEN&gt;" -X POST /trigger/release</screen>
```
Comment (tomschr) [inline xml/obs_authorization_token.xml:96]: Same here, combine all the `screen`s and introduce `replaceable`.

Is `--opertion` a typo? Shouldn't it be named `--operation`?

**Review by tomschr — CHANGES_REQUESTED** [review]
@adrianschroeter Here are some suggestions. :wink: Hope you find them useful.

File: xml/obs_authorization_token.xml [hunk 4]
```diff
@@ -84,6 +84,16 @@
    <screen><command>osc</command> token --create &lt;PROJECT&gt; &lt;PACKAGE&gt;</screen>
    <screen><command>osc</command> token --trigger &lt;TOKEN&gt;</screen>
    <screen><command>curl</command> -H "Authorization: Token &lt;TOKEN&gt;" -X POST /trigger/runservice</screen>
+
+   <para>A token to rebuild a package can be created and trigger by </para>
```
[the remark this answers is not in the document]
Comment (adrianschroeter) [reply xml/obs_authorization_token.xml:88]: rebuild is the name of the action, so I would not change that.

File: xml/obs_authorization_token.xml (hunk 3 above)
Comment (adrianschroeter) [reply xml/obs_authorization_token.xml:96]: yes...

File: xml/obs_authorization_token.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (tomschr) [reply xml/obs_authorization_token.xml:88]: Ahh, right! Sure, in that case, leave it as it is.

File: xml/obs_authorization_token.xml [hunk 5]
```diff
@@ -50,11 +50,11 @@
...
   </sect2>
   <sect2>
-   <title>Executing a Source Service</title>
-   <para> A token can be used to execute a source service. The source service
-    has to be setup for the package first, check the source service chapter for
-    this. A typical example is to update sources of a package from git. A
-    source service for that can be setup with:</para>
+   <title>Executing an action</title>
+   <para> A token can be used to execute specific operations. This can be a 
+	   source service trigger, a rebuild call or release action atm. The setup needs
```
[the remark this answers is not in the document]
Comment (sknorr) [reply xml/obs_authorization_token.xml]: "at the moment" -- but i think the sentence looks better without it.

[the remark this answers is not in the document]
Comment (adrianschroeter) [reply xml/obs_authorization_token.xml]: removed

**Pull request merged**